runtime/lua/vim/secure.lua

```diff
@@ -86,21 +110,77 @@ function M.read(path)
     trust[fullpath] = hash
   end
 
+  write_trust(trust)
+
+  return contents
+end
+
+--- Update the trust status of file at path in the trust database at
```

Comment:
```suggestion
--- Update the trust status of file at {path} in the trust database at
```

Reply:
Resolved in 3e1c77157dfc217919a6b620ac3eb0e7e69b64b4